WalletWasabi/Bases/OctetSerializableBase.cs

```diff
@@ -28,7 +28,7 @@ protected OctetSerializableBase()
 
 		public string ToHex() => ToHex(false);
 
-		public string ToHex(bool xhhSyntax = false)
+		public string ToHex(bool xhhSyntax)
```

Comment:
Could you reverse this change? This xhhSyntax is not something someone should know about.